src/components/Api.js

```diff
@@ -0,0 +1,92 @@
+export default class Api {
+  constructor(data) {
+    this._url = data.baseUrl;
+    this._headers = data.headers;
+    this._avatar = data.avatar;
+  }
+  _checkResponse(res) {
+    if (res.ok) {
+      return res.json();
+    }
+    // if the server returns an error, reject the promise
+    return Promise.reject(`Error: ${res.status}`);
+  }
+
+  async getInitialCards() {
+    const res = await fetch(
+      "https://around-api.en.tripleten-services.com/v1/cards",
```

Comment:
According to the task you need to pass `baseUrl` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `this._baseUrl` (not to duplicate the code)